backend/src/app/services/site/site.service.ts

```diff
@@ -265,6 +266,45 @@ export class SiteService {
     return response;
   }
 
+  async mapSearch(searchTerm = '') {
```

Comment:
I plopped this into Site service, but if anyone has strong objections, I can also move it into its own separate service. Let me know what you think!

Reply:
I think searching for sites belongs in the site service personally.